Calculator/CalculatorTests/CalculatorTests.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  CalculatorTests.swift
+//  CalculatorTests
+//
+//  Created by Whales on 2023/05/31.
+//
+
+import XCTest
+@testable import Calculator
+
+final class CalculatorTests: XCTestCase {
+    var sut: CalculatorItemQueue<Int>!
+    
+    override func setUpWithError() throws {
+        sut = CalculatorItemQueue()
+    }
+    
+    override func tearDownWithError() throws {
+        sut = nil
+    }
+    
+    func test_enqueue함수호출시_배열의원소개수가_늘어나는지() {
+        // given
+        let input = 3
+        let expectation = 1
+        // when
+        sut.enqueue(input)
+        // then
+        XCTAssertEqual(sut.count, expectation)
+    }
+    
+    func test_enqueue함수호출시_배열의원소가_늘어나는지() {
+        // given
+        sut.enqueue(2)
+        let input = 3
+        let expectation = [2, 3]
+        // when
+        sut.enqueue(input)
+        // then
```

Comment:
로직 자체가 간단하기 떄문에 굳이 요런식으로 작성할 필요 없이 바로 데이터를 입력해주는게 조금 더 읽기 쉬워보이네요.
그리고, 2는 바로 인큐하고, 3은 상수로 할당한 후에 인큐하는 방식은 컨벤션의 통일성이 없어서 가독성이 매우 떨어지는것 같습니다. 🥲
```suggestion
        // given
        let expectation = [2, 3]
        // when
        sut.enqueue(2)
        sut.enqueue(3)
        // then
```

Reply:
테스트 코드를 짤 때 배열의 은닉화를 풀고 배열 = [2] 로 직접 넣어준 뒤, 3을 넣었을 때 앞으로 들어가는게 아니라 뒤로 들어간다는 걸 눈으로 확인하고 싶어서 만들었던 코드였습니다.
근데 은닉화를 다시 해주면서 테스트 코드에 나는 오류부분을 고치다보니 저기 한 줄만 후루룩 고쳐버렸네요ㅠㅠ
앞으로 코드를 짜거나 수정을 할 때 이런 부분들을 좀 더 신경 써 보겠습니다!!
감사합니다 🙇🏻‍♀️